torchci/pages/kpis.tsx

```diff
@@ -0,0 +1,45 @@
+import { Grid } from "@mui/material";
+import TimeSeriesPanel from "components/metrics/panels/TimeSeriesPanel";
+import dayjs from "dayjs";
+import { RocksetParam } from "lib/rockset";
+import { useState } from "react";
+
+const ROW_HEIGHT = 340;
+
+export default function Kpis() {
+    const [startTime, setStartTime] = useState(dayjs().startOf("year").add(1, 'month'));
```

Comment:
why add a month?

Reply:
Whoops. Some copy/pasta that snuck in.  It's gone now